docs/source/getting-started.mdx

```diff
@@ -119,6 +119,8 @@ The above `start` script tells TypeScript to compile your code into JavaScript b
 
 #### Set up with JavaScript
 
+> All code examples in this getting started guide are in Typescript. You must transpile them to JavaScript.
```

Comment:
TypeScript has a capital S.

This is helpful, but I was referring to something that I now realize is not easily visible from reading the Markdown file: the `Disclaimer` component.

Reply:
Fixing the typo.

Thanks for pointing out the `Disclaimer`, it's being fixed in https://github.com/apollographql/docs-rewrite/pull/282